views/layout.pug

```diff
@@ -2,6 +2,38 @@ doctype html
 html
   head
     title= title
+    meta(content='width=device-width, initial-scale=1, user-scalable=no', name='viewport')
+
+    link(rel='stylesheet', href='//fonts.googleapis.com/css?family=Montserrat')
     link(rel='stylesheet', href='/stylesheets/style.css')
+    link(rel='stylesheet', href='//maxcdn.bootstrapcdn.com/bootswatch/3.3.6/cyborg/bootstrap.min.css')
+    link(rel='stylesheet', href='//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css')
   body
-    block content
+    .navbar.navbar-default.navbar-fixed-top
+      .container
+        .navbar-header
+          a.navbar-brand(href='../')
+            i.fa.fa-paper-plane
+            | &nbsp; N O T I F Y
```

Comment:
Thanks, I totally missed this one. As a matter of fact I already have a CSS class called `spaced` which will take care of this spacing. Thanks for notice it.